opsgenie/integration_util.go

```diff
@@ -33,9 +33,9 @@ func expandOpsgenieIntegrationResponders(d *schema.ResourceData) []integration.R
 
 func validateOpsgenieIntegrationName(v interface{}, k string) (ws []string, errors []error) {
 	value := v.(string)
-	if !regexp.MustCompile(`^[a-zA-Z0-9_-]+$`).MatchString(value) {
```

Comment:
this line is not correct and not a true statement in terms of regex also throws compilation error